data/map planets.txt

```diff
@@ -249,12 +249,24 @@ planet "Ashy Reach"
 	"required reputation" 15
 	security 0.1
 
+planet Asphodel
+	attributes uninhabited
+	landscape land/badlands3
+	description `Despite being nominally in the habitable zone, this planet's surface is continually scorched by flares from the system's stars, steadily blasting away its atmosphere. Although some small research stations are present in orbit, providing valuable insights into the nature and behavior of flare stars, the researchers generally avoid descending to the planet's surface unnecessarily.`
```

Comment:
```suggestion
	description `Despite being nominally in the habitable zone, this planet's surface is continually scorched by Gacrux and its solar flares, steadily blasting away its atmosphere. The Republic has put some small research stations in orbit around Asphodel, and while they provide valuable insights into the nature and behavior of flare stars, the researchers generally avoid descending to the planet's surface unnecessarily. None of the stations appear to have the facilities to service your ship, however.`
```
Not sure if Gacrux has multiple stars in real life, from a search it says it's debated whether it has a white dwarf companion to the confirmed red giant.

Maybe instead of the Republic, could be the Deep that funds the stations, or Bounty / Heartland / some university from Chiron or the Paradise Planets.

Reply:
I based this off the stars that are there in-game (a binary of a k8 and an m5), not what's there in real life.

Reply:
> Maybe instead of the Republic, could be the Deep that funds the stations, or Bounty / Heartland / some university from Chiron or the Paradise Planets.

I do like the idea of mentioning one of the universities on Chiron here, and that does seem appropriate.

Reply:
Yeah fair. ~~I mean it could get changed, or...~~
```suggestion
	description `Despite being nominally in the habitable zone, this planet's surface is continually scorched by the stars of Gacrux and their solar flares, steadily blasting away its atmosphere. The Republic has put some small research stations in orbit around Asphodel, and while they provide valuable insights into the nature and behavior of flare stars, the researchers generally avoid descending to the planet's surface unnecessarily. None of the stations appear to have the facilities to service your ship, however.`
```